frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/SettingsUpdatesForm.unit.spec.js

```diff
@@ -17,47 +22,56 @@ const render = () => {
 
 describe("SettingsUpdatesForm", () => {
   it("shows custom message for Cloud installations", () => {
-    const isHostedSpy = jest.spyOn(MetabaseSettings, "isHosted");
-    isHostedSpy.mockImplementation(() => true);
+    mockSettings({ "is-hosted?": true });
 
     render();
     screen.getByText(/Metabase Cloud keeps your instance up-to-date/);
-
-    isHostedSpy.mockRestore();
   });
 
   it("shows correct message when latest version is installed", () => {
-    const versionIsLatestSpy = jest.spyOn(MetabaseSettings, "versionIsLatest");
-    versionIsLatestSpy.mockImplementation(() => true);
+    mockSettings({
+      version: createMockVersion({ tag: "v1.0.0" }),
+      "version-info": createMockVersionInfo({
+        latest: createMockVersionInfoRecord({ version: "v1.0.0" }),
+      }),
+    });
 
     render();
     screen.getByText(/which is the latest and greatest/);
-
-    versionIsLatestSpy.mockRestore();
   });
 
   it("shows correct message when no version checks have been run", () => {
+    mockSettings({
```

Comment:
what would happen if we don't `mockSettings` here. I couldn't see any obvious reason why we need to have a mock here since the previous test stays relatively the same except this new mock settings.

Reply:
I suppose it'd pass, but I personally like it when settings are defined explicitly here. We're testing a case when version info isn't yet loaded and just relying on the fact that `MetabaseSettings` values should be empty can be fragile IMO